plugin.php

```diff
@@ -320,6 +321,109 @@ function create_initial_json_routes() {
 			'context'          => array(),
 		)
 	));
+
+	/**
+	 * Comments
+	 */
+	$controller = new WP_JSON_Comments_Controller;
+	register_json_route( 'wp', '/comments', array(
+		array(
+			'methods'   => WP_JSON_Server::READABLE,
+			'callback'  => array( $controller, 'get_items' ),
+			'args'      => array(
+				'post_id'      => array(
+					'default'      => null,
+				),
+				'user_id'      => array(
+					'default'      => 0,
+				),
+				'per_page'     => array(
+					'default'      => 10,
+				),
+				'page'         => array(
+					'default'      => 1,
+				),
+				'status'       => array(
+					'default'      => 'approve',
+				),
+				'type'         => array(
+					'default'      => 'comment',
+				),
+				'parent_id'    => array(),
+				'search'       => array(),
+				'order'        => array(
+					'default'      => 'DESC',
+				),
+				'orderby'      => array(
+					'default'      => 'comment_date_gmt',
```

Comment:
This is strictly a `WP_Comment_Query` parameter.  If we convert parameter names here, we would have to do that for all of the endpoints.  I would rather remove it.